FileOrganization.md

```diff
@@ -153,19 +153,21 @@ class FruitSellerView: UIView {
 
     // good: split rotation and fade animation constants into different groups
 
-    private struct RotationAnimationConstants {
+    private enum RotationAnimationConstants {
         static let angle: CGFloat = .pi / 45.0
         static let delay: CGFloat = 0.15
         static let duration: CGFloat = 0.3
         static let speed: CGFloat = 100
         static let identifier: String = "FruitSellerViewRotationAnimationIdentifier"
     }
 
-    private struct FadeAnimationConstants {
+    private enum FadeAnimationConstants {
         static let delay: CGFloat = 0.1
         static let duration: CGFloat = 0.25
         static let speed: CGFloat = 200
         static let identifier: String = "FruitSellerViewFadeAnimationIdentifier"
     }
 }
 ```
+
+Note: We use `enum` rather than `struct` as it can help reduce potential confusion. Running `let x = RotatationAnimationConstants()` would be valid code if we were to use a `struct`, for an `enum` it will throw an error. 
```

Comment:
Trying to brainstorm if there are other conventions related to enums so that we can group them and have a dedicated page on enums as well.

Reply:
We have nothing specific in Outlook. Just what is in the Swift Style Guide.